docs/index.md

```diff
@@ -0,0 +1,45 @@
+---
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: page
+---
+<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
+<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
+<script type="text/x-mathjax-config">
+        MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ["\\(","\\)"]], displayMath: [ ["$$","$$"], ["\\[","\\]"] ]
+        }});
+</script>
+
+<link rel="stylesheet" href="assets/main.css">
+
+<font size="+4"><p style="text-align: center;">The Dafny Language and Software Verification System</p></font>
```

Comment:
`<font>` has been deprecated for years; let's not use that.  Should this be an `<h1>`?
Ideally though it would be best to stick with Markdown; we can give the title an ID and style it with CSS

Reply:
Corrected. (the html came from a page-generating tool that we are no longer using)